re/README.md

```diff
@@ -0,0 +1,67 @@
+# Re
+
+Re service provides a barebones HTTP API for development of a Mainflux
+service.
+
+## Configuration
+
+The service is configured using the environment variables presented in the
+following table. Note that any unset variables will be replaced with their
+default values.
+
+| Variable              | Description                                             | Default |
+|-----------------------|---------------------------------------------------------|---------|
+| MF_RE_LOG_LEVEL   | Log level for re service (debug, info, warn, error) | error   |
+| MF_RE_HTTP_PORT   | Re service HTTP port                                | 9021    |
+| MF_RE_SERVER_CERT | Path to server certificate in pem format                |         |
+| MF_RE_SERVER_KEY  | Path to server key in pem format                        |         |
+| MF_JAEGER_URL         | Jaeger server URL                                       |         |
+| MF_RE_SECRET      | Re service secret                                   | secret  |
+
+## Deployment
+
+The service itself is distributed as Docker container. The following snippet
+provides a compose file template that can be used to deploy the service container
+locally:
+
+```yaml
+version: "3"
+services:
+  re:
+    image: mainflux/re:[version]
```

Comment:
you can provide a link to the section in docker-compose, no need to duplicate code